src/01-general/work-orders.apib

```diff
@@ -0,0 +1,18 @@
+## Work Orders [/workOrders]
+
+ ### workOrders.draft [POST /workOrders.draft]
```

Comment:
B: spaces at the start of all lines (causing weird issues in the compiled file). The request is not recognized as such.